make/modules/java.base/Lib.gmk

```diff
@@ -87,6 +89,7 @@ $(eval $(call SetupJdkLibrary, BUILD_LIBNIO, \
         -framework CoreFoundation \
         -framework CoreServices, \
     LIBS_windows := advapi32.lib mswsock.lib ws2_32.lib, \
+    LIBS_bsd := -pthread, \
```

Comment:
Not that it is a super big deal, but I've tried to organize stanzas like this so that the order of OS:es is consistent across files, and so that the unixes go together. This makes it easier to spot patterns, and see when things can be generalized to a "unix" os type level. 

Basically, my sort order has been:
* unix (in general)
* linux
* solaris (when it was present)
* aix
* macosx
* windows

My recommendation is that you try to stick bsd in after aix and before macosx, since it is very much a pure unix, but still share some parts of macosx (while the latter still has a lot of own special stuff, like all the system frameworks etc).

Reply:
That makes sense. For this overview I've tries to keep the order as close as I could to the current out-of-tree port, but I agree your order makes more sense.

Reply:
Also, this should use the new $(LIBPTHREAD).